src/sass/ResultActionMenu.scss

```diff
@@ -122,6 +123,11 @@ $tooltip-z-index: 1000;
     }
 }
 
+//Fix for attach icon aligment
+.CoveoAttachResult div.coveo-icon-attach{
```

Comment:
Seems odd?

Reply:
I did not see the transparent border came from the %icon tag at line 15. I propose to remove the transparent border there since it is causing a 1 pixel shift with the Quickview Icon that does not have that transparent border